infra-as-code/bicep/openai.bicep

```diff
@@ -34,7 +34,6 @@ resource openAiAccount 'Microsoft.CognitiveServices/accounts@2023-10-01-preview'
     name: 'blocking-filter'
     properties: {
       #disable-next-line BCP037
-      type: 'UserManaged'

```

Comment:
Why was this removed? Is it no longer part of the API?

Reply:
If you're going to remove this, then you need to remove the `#disable-next-line BCP037`.  Which is now applying to `basePolicyName: 'Microsoft.Default'`